interfaces/types.tsx

```diff
@@ -0,0 +1,26 @@
+// Constants Interfaces
+
+export interface Crypto {
+    id: number,
+    name: string,
+    initials: string,
+    image: string,
+    value: number,
+    metric: number,
+    icon: string,
+    color: string,
+}
+
+// Actions Interfaces
```

Comment:
[minor] Refrain from leave unnecesary comments on your code; depending on the situation, you might want to clarify certain hard to understand pieces of code